src/main/java/sirius/biz/jupiter/Jupiter.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.jupiter;
+
+import redis.clients.jedis.util.SafeEncoder;
+import sirius.db.redis.Redis;
+import sirius.db.redis.RedisDB;
+import sirius.kernel.Sirius;
+import sirius.kernel.commons.Value;
+import sirius.kernel.commons.Values;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.health.Average;
+import sirius.kernel.health.Log;
+
+import javax.annotation.Nullable;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+/**
+ * Provides a facility to connect to one or more <a href="https://github.com/scireum/jupiter">Jupiter</a> instances.
+ * <p>
+ * As Jupiter speaks the RESP protocol as defined by Redis, we actually use the connection handling as provided
+ * by {@link Redis}. However, since Jupiter is commonly "stateless" we permit to define a fallback connection
+ * in case the main host goes down.
+ * <p>
+ * The config is provided via <tt>redis.pools.jupiter</tt> as well as the <tt>jupiter.settings.[name]</tt> section in
+ * the config.
+ */
+@Register(classes = Jupiter.class, framework = Jupiter.FRAMEWORK_JUITER)
+public class Jupiter {
+
+    /**
+     * Specifies the framework to enable when using Jupiter.
+     */
+    public static final String FRAMEWORK_JUITER = "jupiter";
+
+    /**
+     * Determines the name of the logger used for all Jupiter related logging.
+     */
+    public static final Log LOG = Log.get("jupiter");
```

Comment:
```suggestion
    public static final Log LOG = Log.get(FRAMEWORK_JUPITER);
```
I think this makes sense(?)

Reply:
As these are semantically different - I'd rather repeat the name here again